lib/author.rb

```diff
@@ -0,0 +1,39 @@
+class Author
+  attr_reader :first_name,
```

Comment:
Since we never call `artist.first_name` or `artist.last_name` we don't need attr_readers for these.